src/components/slider/slider.tsx

```diff
@@ -175,7 +176,7 @@ export class Slider {
                         aria-valuenow={this.multiplyByFactor(this.getValue())}
                         aria-label={this.label}
                         aria-disabled={this.disabled || this.readonly}
-                        data-step={this.step}
+                        data-step={this.multiplyByFactor(this.step)}
```

Comment:
Before we merge a breaking change, I just want to get a second opinion from @jgroth: do you agree that it's reasonable that `step` should also be multiplied by `factor`?

Example:

We have a slider that goes from 0 to 100. The value is represented as a decimal number, from 0 to 1. Thus, we have a `factor` of `100`. If we want the slider to use steps of 5 in the UI, corresponding to stepping the value in multiples of 0.05, before this PR we'd set `step={5}`:

```tsx
this.value = 0.3 // Visualised by the slider as "30"

<limel-slider
	value={this.value}
	valuemin={0}
	valuemax={1}
	factor={100}
	step={5}
/>
```

After this PR, we'd instead set `step={0.05}`:

```tsx
this.value = 0.3 // Visualised by the slider as "30"

<limel-slider
	value={this.value}
	valuemin={0}
	valuemax={1}
	factor={100}
	step={0.05}
/>
```

Of course, how reasonable this looks may depend somewhat on where the `factor={100}` is placed 😄:

```tsx
<limel-slider
	value={this.value}
	valuemin={0}
	valuemax={1}
	step={5}
	factor={100}
/>
```

vs

```tsx
<limel-slider
	value={this.value}
	valuemin={0}
	valuemax={1}
	step={0.05}
	factor={100}
/>
```

Reply:
Yeah I agree, the old behavior was weird, having a step of `5` for a value between `0` and `1` does not make sense :smile: 
I'm not sure I agree on the breaking change though. Sure, there is a change in behavior but that can be said for almost any bug fix

Reply:
I'll copy my reply to Bef when he asked about that:

> It's breaking for anyone already using the slider with multiplier and step if they already included the multiplier in the step themselves.
>
> `<limel-slider value={this.value} multiplier="100" step="5" />` for example. When this fix is released, they'd have to change that code to `<limel-slider value={this.value} multiplier="100" step="0.05" />`.

The [Semantic Versioning docs](https://semver.org/) says (my _cursive_):

> Given a version number MAJOR.MINOR.PATCH, increment the:
>
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards compatible manner, and
> - PATCH version when you make _backwards compatible bug fixes_.

I don't think this can be considered a backwards compatible bug fix 🤷‍♂️

Reply:
> Yeah I agree, the old behavior was weird, having a step of 5 for a value between 0 and 1 does not make sense 😄

Oh, and thanks for the confirmation! I think that's the most important, because whether we do a major bump or a patch bump, I wouldn't want to merge it an only _then_ hear a very convincing argument from you about why this change doesn't make any sense 😅🙈